README.md

```diff
@@ -261,6 +261,12 @@ In order to run tests with both Redis and Redis Cluster, you can run the test pr
 ./build/test/test_redis++ -h host -p port -a auth -n cluster_node -c cluster_port
 ```
 
+Alternatively you can provide a redis connection uri with -u, e.g. "redis://127.0.0.1:6379":
```

Comment:
The test program cannot use URI to specify parameters. Because the program can test with both standalone Redis and Redis Cluster in a single test. However, the URI can only specify either standalone Redis or Redis Cluster.